azure_functions_worker/dispatcher.py

```diff
@@ -426,6 +431,14 @@ def _serialize_exception(self, exc):
                 request_id=self.request_id,
                 function_environment_reload_response=failure_response)
 
+    @disable_feature_by(constants.PYTHON_ROLLBACK_CWD_PATH)
```

Comment:
This function will not executed when PYTHON_ROLLBACK_CWD_PATH is set to `true` or `1`.